CMakeLists.txt

```diff
@@ -35,12 +35,12 @@ else()
     find_package(Lua)
 endif()
 
-find_package(Boost 1.53.0 COMPONENTS system iostreams REQUIRED)
+find_package(Boost 1.58.0 COMPONENTS system iostreams REQUIRED)
 
 add_subdirectory(src)
 add_executable(tfs ${tfs_SRC})
 
-include_directories(${MYSQL_INCLUDE_DIR} ${LUA_INCLUDE_DIR} ${Boost_INCLUDE_DIRS} ${PUGIXML_INCLUDE_DIR} ${GMP_INCLUDE_DIR})
+include_directories(${CMAKE_CURRENT_SOURCE_DIR}/src ${MYSQL_INCLUDE_DIR} ${LUA_INCLUDE_DIR} ${Boost_INCLUDE_DIRS} ${PUGIXML_INCLUDE_DIR} ${GMP_INCLUDE_DIR} ${CMAKE_CURRENT_LIST_DIR}/ext/Beast/include/)
```

Comment:
The problem is that without this I would have to use ```#include "../otpch.h"``` which breaks horribly in MSVC. I have no idea how to work around this in a different way.